architecture-building-systems/CityEnergyAnalyst
# I2182 read occupancy schedules

This addresses issue #2182.
As discussed with @daren-thomas, the first step towards user defined occupancy schedules, is to make the occupancy model read (user-defined) schedules without changing too much of the code.
This PR does exactly this.
- during execution of the demand calculation it checks if there is a file `{building_name}_schedules.csv` is present in `inputs/building_properties`.
- if yes, it will import these schedules for the calculation
- if not, it will create the schedules with @martin-mosteiro's occupancy model and save them to a csv file

This file could then be modified by the user (in excel) for a second run or for other scenarios (before we have more sophisticated tools).

How to test:
(1) run the demand of any scenario -> the script should create schedule files in your input folder
(2) keep some of the files and delete others
(3) re-run the demand -> now the script should only create the missing files
